ccmlib/scylla_node.py

```diff
@@ -563,7 +569,10 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
             args += ['--overprovisioned']
         if '--prometheus-address' not in args:
             args += ['--prometheus-address', data['api_address']]
-        if replace_address:
+        if replace_node:
+            assert replace_address is None, "replace_node and replace_address cannot be specified together"
+            args += ['--replace-node-first-boot', replace_node]
```

Comment:
what doesn't first-boot mean, would anyone thing to call it in a different context ? i.e. passing it not on first boot ? 

and it's a bit weird that this commit doesn't say host_id anywhere... i.e. `replace_node` is actually `replace_node_host_id`, i.e. nothing here indicate what's need to be passing down.

Reply:
It basically follows the convention of --replace-endpoint-first-boot.
The details are in the respective documentation that was updated in https://github.com/scylladb/scylladb/pull/12316.